addons/crm/res_config_view.xml

```diff
@@ -32,23 +32,32 @@
                             </div>
                         </div>
                     </group>
-	            </div>
-                <xpath expr="//p[@name='config_email_integration']" position="after">
-                    <group name="default_alias">
-                        <label for="id" string="Leads Email Alias"/>
-                        <div attrs="{'invisible': [('alias_domain', '=', False)]}">
+                </div>
+                <div name="config_sale" position="after">
+                    <div name="config_alias">
+                        <separator string="Emails Integration"/>
+                        <p name="config_email_integration">
+                            Odoo allows to automatically create leads (or others documents)
+                            from incoming emails.
+                        </p> 
+                        <group>
+                            <label for="id" string="Leads Email Alias"/>
                             <div>
-                                <field name="alias_prefix" class="oe_inline" attrs="{'required': [('alias_domain', '!=', False)]}"/>
-                                @
-                                <field name="alias_domain" class="oe_inline" readonly="1"/>
+                                <div>
+                                    <span attrs="{'invisible': [('group_multi_salesteams', '!=', True)]}">
+                                        <label string="Each sales team has an email alias that can be defined on the sales team form view" class="oe_inline oe_form_label_help oe_align_right"/>
+                                    </span>
+                                    <span class="oe_inline" attrs="{'invisible': [('group_multi_salesteams', '!=', False)]}">
+                                        <label string="Automatically create new leads when an email is sent to the following alias :" class="oe_inline oe_form_label_help oe_align_right"/>
+                                        <field name="alias_prefix" class="oe_inline"/>
+                                        <span>@</span>
+                                        <field name="alias_domain"  class="oe_inline" readonly="1"/>
```

Comment:
By the way, if the user has not configured the alias domain he will have some issues. And he cannot modify it here, it is configured in the general settings. Maybe add a link to the general settings if the alias domain is not set ?